test-butler-app/src/main/java/com/linkedin/android/testbutler/SystemLocaleChanger.java

```diff
@@ -0,0 +1,24 @@
+package com.linkedin.android.testbutler;
+
+import java.util.Locale;
+
+class SystemLocaleChanger {
+    boolean setSystemLocale(String language, String country) {
+        return setSystemLocale(new Locale(language, country));
+    }
+
+    boolean setSystemLocale(Locale locale) {
```

Comment:
I did read their comment and it makes sense that there might be problems with Espresso. The thing with our automation is that we don't have access to the application context so we can't use their approach. I figured it was reasonable to separate this out by explicitly denoting it as `SystemLocaleChanger` since it changes the system locale instead of the app context locale. Also, the apps that use their locale changer have to honor that change and there may be cases where that is not honored in the app.